src/SFML/Graphics/Shape.cpp

```diff
@@ -204,7 +204,6 @@ void Shape::update()
     // Texture coordinates
     updateTexCoords();
 
-    // Outline
```

Comment:
It seemed kinda redundant. Since the method it self is named updateOutline, but I can undo that If needed.